R/convertIRF.R

```diff
@@ -0,0 +1,29 @@
+#' convertIRF
+#'
+#' Add missing periods to data frame and interpolate missing values.
+#' The ISO codes of "IRF" data are compared with the official ISO code country list.
+#' NA values are replaced with zeros and historical data are corrected.
+#'
+#' @param x MAgPIE object.
+#'
+#' @return The "IRF" data with spatial entries for each country.
+#'
+#' @author Anastasis Giannousakis, Fotis Sioutas
+#'
+#' @examples
+#' \dontrun{
+#' a <- readSource("IRF", convert = TRUE)
+#' }
+#'
+#' @importFrom quitte as.quitte
+#' @importFrom quitte interpolate_missing_periods
```

Comment:
no need for a second line for the quitte imports, they can all be in the same line separated by spaces

Reply:
ok